tests/unit/test_config.py

```diff
@@ -554,11 +553,8 @@ def test_master_file_roots_glob(self, tempdir, fpath):
             )
         config = sconfig.master_config(fpath)
         base = config['file_roots']['base']
-        self.assertEqual(set(base), set([
-            os.path.join(tempdir, 'a'),
-            os.path.join(tempdir, 'b'),
-            os.path.join(tempdir, 'c')
-        ]))
+        self.assertEqual(set(base),
+                         {os.path.join(tempdir, 'a'), os.path.join(tempdir, 'b'), os.path.join(tempdir, 'c')})
```

Comment:
Any clue why all of these blocks changed?

I don't *think* I touched any of these :suspect: